README.md

```diff
@@ -141,6 +141,26 @@ and pierre.casteran@labri.fr.
   patches from Coq developers when they introduce a breaking change (this is
   particularly recommended for plugins).
 
+- **What license to choose for a coq-community project?**
+
+  The only strict requirement is to use an open source license.
+  However, if you create a new project or propose to transfer a
+  project of which you are the sole copyright owner, we strongly
+  encourage you to (re)license your project under one of the following
+  two licenses:
+
+  - [MIT license](https://choosealicense.com/licenses/mit/): the most
+    permissive and most popular open source license.  This is the best
```

Comment:
Is MIT really the *most permissive* open source license? I thought Unlicense counted as open source...

Reply:
Indeed, and the Unlicense is not the only exception: the Boost license is also more permissive. I'll change "the most" to "a very".